### PR TITLE
Add support to view rewards by day

### PR DESCRIPTION
**Pull request by juandavidkincaid**
[DEW-91](https://constellationnetwork.atlassian.net/browse/DEW-91)

### Changes
+ Updated total rewards endpoint path name.
+ Added DAG rewards tab in address.
+ Grouped rewards by day.

[DEW-91]: https://constellationnetwork.atlassian.net/browse/DEW-91?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/RewardsTable/RewardRow.tsx [hunk 1]
```diff
@@ -65,6 +65,8 @@ export const RewardRow = ({
         <div className={clsx(isLastRow ? styles.txnCellLastRow : styles.txnCell)}>
           {reward.metagraphId ? (
             <Link to={`/metagraphs/${reward.metagraphId}/snapshots/${reward.ordinal}`}>{reward.ordinal}</Link>
+          ) : reward.ordinal === -1 ? (
```
Comment (AlexBrandes) [inline src/components/RewardsTable/RewardRow.tsx:68]: What case does this represent? Why would a reward have a -1 ordinal?

File: .github/workflows/deploy-staging.tmp.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: Staging Tmp Build
+on:
+  push:
+    branches:
+      - development/juan/DEW-91,LE-792/change-total-rewards-endpoint-path
```
Comment (AlexBrandes) [inline .github/workflows/deploy-staging.tmp.yml:5]: Shouldn't this just be `deploy/staging`?

File: src/components/RewardsTable/RewardRow.tsx (hunk 1 above)
Comment (juandavidkincaid) [reply src/components/RewardsTable/RewardRow.tsx:68]: This represents rewards grouped by day, they don't have an ordinal to check, but we can use rewards records directly, the API supports both.

File: .github/workflows/deploy-staging.tmp.yml (hunk 2 above)
Comment (juandavidkincaid) [reply .github/workflows/deploy-staging.tmp.yml:5]: Nope, this file exists just for this PR/branch will be deleted before merging.

**Pull request merged**